CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
-# Upcoming (v0.5.11)
+# Upcoming (v0.5.12)
```

Comment:
Shouldn't this be shifted?

Reply:
As in, first the version and then the data (upcoming)

Reply:
Yeah that would probably make more sense tbh